facebook.js

```diff
@@ -114,12 +115,12 @@ const show_facebook_cv_tags = function() {
 
 const observer = new MutationObserver(function(mutations) {
     mutations.forEach(function(mutation) {
-        show_facebook_cv_tags();
+        setTimeout(function(){show_facebook_cv_tags(mutation.target);},1500);
```

Comment:
when the node is injected in the DOM, the img is not available in the same time, you have to wait a bit to have the img loaded, mainly is for theatre mode, when you see an album etc..